phoenix-core/src/it/java/org/apache/phoenix/execute/PartialCommitIT.java

```diff
@@ -259,10 +259,10 @@ private PhoenixConnection getConnectionWithTableOrderPreservingMutationState() t
         Connection con = driver.connect(url, new Properties());
         PhoenixConnection phxCon = new PhoenixConnection(con.unwrap(PhoenixConnection.class));
         final Map<TableRef,Map<ImmutableBytesPtr,MutationState.RowMutationState>> mutations = Maps.newTreeMap(new TableRefComparator());
-        return new PhoenixConnection(phxCon) {
+        return new PhoenixConnection(phxCon, null) {
```

Comment:
Probably a good idea to still have a PhoenixConnection(PhoenixConnection conn) copy constructor that just calls the other one with a null second arg (and put that in a separate pull request).